README.md

```diff
@@ -100,7 +100,8 @@ subview.addSubview(subsub2view)
 
 ## AnchorsBuilder
 
-`AnchorsBuilder` is DSL builder of `Anchors` for making autolayout constraint between views or view itself. most are used within the `anchors` function of Layout.
+`AnchorsBuilder`` is a DSL builder for Anchors types that aids in the creation of autolayout constraints between views.
+It is mainly used within anchors, a method of Layout.
```

Comment:
Strange code highlighting of "Anchors" and "anchors" here

Reply:
Thank you. I'll fix it right away.